<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTabWidget" name="thickness_tabWidget">
     <widget class="NumericParameterWidget" name="Thickness">
      <attribute name="title">
       <string>Thickness</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTabWidget" name="other_tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="NumericParameterWidget" name="NSLDR">
      <attribute name="title">
       <string>NSLD (real)</string>
      </attribute>
     </widget>
     <widget class="NumericParameterWidget" name="NSLDI">
      <attribute name="title">
       <string>NSLD (imag)</string>
      </attribute>
     </widget>
     <widget class="NumericParameterWidget" name="MSLD_rho">
      <attribute name="title">
       <string>MSLD rho</string>
      </attribute>
     </widget>
     <widget class="NumericParameterWidget" name="MSLD_theta">
      <attribute name="title">
       <string>MSLD theta</string>
      </attribute>
     </widget>
     <widget class="NumericParameterWidget" name="MSLD_phi">
      <attribute name="title">
       <string>MSLD phi</string>
      </attribute>
     </widget>
     <widget class="NumericParameterWidget" name="Roughness">
      <attribute name="title">
       <string>Roughness</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Name_label">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Layer name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Name_lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="roughness_model_label">
       <property name="text">
        <string>Roughness model:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="roughness_model_comboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="sublayers_label">
       <property name="text">
        <string>   Sublayers:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sublayers_spinBox">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NumericParameterWidget</class>
   <extends>QWidget</extends>
   <header location="global">NumericParameterWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>Name_lineEdit</tabstop>
  <tabstop>thickness_tabWidget</tabstop>
  <tabstop>other_tabWidget</tabstop>
  <tabstop>roughness_model_comboBox</tabstop>
  <tabstop>sublayers_spinBox</tabstop>
  <tabstop>Name_label</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>other_tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>show_hide_roughness_extras()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>401</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_hide_roughness_extras()</slot>
 </slots>
</ui>
